test/integration/zenodo_datapackage_test.py

```diff
@@ -8,24 +8,19 @@
 
 
 class TestZenodoDatapackages:
-    """Ensure production & sandbox Datastores point to valid datapackages."""
+    """Ensure all DOIs in Datastore point to valid datapackages."""
 
     @pytest.mark.xfail(
-        raises=(MaxRetryError, ConnectionError, RetryError, ResponseError)
-    )
-    def test_sandbox_datapackages(self):
-        """All datasets point to valid descriptors with 1 or more resources."""
-        ds = Datastore(sandbox=True)
-        for dataset in ds.get_known_datasets():
-            desc = ds.get_datapackage_descriptor(dataset)
-            assert list(desc.get_resources())
-
-    @pytest.mark.xfail(
-        raises=(MaxRetryError, ConnectionError, RetryError, ResponseError)
+        raises=(
+            MaxRetryError,
```

Comment:
We could mock out the Zenodo interaction, though I guess a lot of the funky logic is "are we using the right access token for the URL for the dataset" so maybe we just keep this around. How flaky does this turn out to be?